ansible_collections/arista/cvp/galaxy.yml

```diff
@@ -7,7 +7,7 @@ namespace: arista
 name: cvp
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 3.2.0
+version: null
```

Comment:
hey @titom73 is there any specific reason why this was set to `null`? I noticed this is causing the collection to appear without a version if you issue the `ansible-galaxy collection list` command
```
[WARNING]: Collection at '/home/avd/arista-ansible/ansible-cvp/ansible_collections/arista/cvp' does 
not have a valid version set, falling back to '*'. Found version: 'None'

# /home/avd/arista-ansible/ansible-cvp/ansible_collections
Collection Version
---------- -------
arista.cvp *      
```